### PR TITLE
Translated titles into korean

### PR DESCRIPTION
**Pull request by sushil97**
Part of #24388 
I have updated following titles to korean :
Glossary
Use the Web UI Dashboard

Files updated : 
content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md
content/ko/docs/reference/glossary/index.md

### CODE REVIEW

File: content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 10
 card:
   name: tasks
   weight: 30
-  title: Use the Web UI Dashboard
+  title: 웹 UI 대시 보드 사용
```
Comment (gochist) [inline content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md:8]:
```suggestion
  title: 웹 UI 대시보드 사용
```
It is better to translate it into the single word according to the [Korean l10n glossary](https://kubernetes.io/ko/docs/contribute/localization_ko/#%ED%95%9C%EA%B8%80%ED%99%94-%EC%9A%A9%EC%96%B4%EC%A7%91).

**Comment by gochist** [comment]
@sushil97 To finish off, please squash those commits into one single commit and rebase it onto dev-1.19-ko.3, current working branch for Korean l10n.

**Comment by sushil97** [comment]
@gochist i think i have done some mistake while squashing as this is my first time doing this. Should i close this pull request ?

**Comment by irvifa** [comment]
@gochist just wondering since there’s part of id localization there. Curious on how usually folks maintain their localization dev branch

**Comment by gochist** [comment]
@sushil97 If upstream is registered as a kubernetes/website repository, you can try as follows. If you have any difficulties, please tell me. I'll do it for you.

```
git fetch upstream
git rebase --onto upstream/dev-1.19-ko.3 HEAD~1
git push --force
```

**Comment by gochist** [comment]
@irvifa Sorry for the inconvenience. When merged, we will not include the id content.

**Comment by irvifa** [comment]
@gochist It’s okay just want to make sure I understand it correctly since I’d like to know more about localization process. Amazing works by the way 🎉

**Comment by sushil97** [comment]
@gochist i tried something similar only but it did'nt went as expected. Can you do it for me?

**Comment by gochist** [comment]
@sushil97 Thank you for your contribution! When team milestone ends, it will be merged into the master and you can check it in live.

/milestone dev-1.19-ko.3
/lgtm
/approve

**Comment by gochist** [comment]
> @gochist It’s okay just want to make sure I understand it correctly since I’d like to know more about localization process. Amazing works by the way

There is a [guide for Korean l10n](https://kubernetes.io/ko/docs/contribute/localization_ko/) briefly explaining the process, but unfortunately, it is in Korean. Though, there is a machine translator.

And the latest milestone progress of Korean l10n team is shared by the team leader, @seokho-son in [the community slack #sig-docs-localization](https://app.slack.com/client/T09NY5SBT/C0191RDKHU1/thread/C0191RDKHU1-1601025948.009300). Through this, I think you can understand a little more about the Korean team's process.

It would be nice to keep discussing this in the slack #sig-docs-localizations channel.

**Comment by gochist** [comment]
@irvifa 
> @gochist It’s okay just want to make sure I understand it correctly since I’d like to know more about localization process. Amazing works by the way

There is a [guide for Korean l10n](https://kubernetes.io/ko/docs/contribute/localization_ko/) briefly explaining the process, but unfortunately, it is in Korean. Though, there is a machine translator.

And the latest milestone progress of Korean l10n team is shared by the team leader, @seokho-son in [the community slack #sig-docs-localization](https://app.slack.com/client/T09NY5SBT/C0191RDKHU1/thread/C0191RDKHU1-1601025948.009300). Through this, I think you can understand a little more about the Korean team's process.

It would be nice to keep discussing this in the slack #sig-docs-localizations channel.

**Pull request merged**